storage/remote/otlptranslator/prometheus/normalize_name_test.go

```diff
@@ -179,13 +179,13 @@ func TestRemoveItem(t *testing.T) {
 	require.Equal(t, []string{"b", "c"}, removeItem([]string{"a", "b", "c"}, "a"))
 }
 
-func TestBuildCompliantNameWithNormalize(t *testing.T) {
+func TestBuildCompliantNameWithSuffixes(t *testing.T) {
```

Comment:
This tests suffix adding mode, confer the inverse which is named `TestBuildCompliantNameWithoutSuffixes`.

Reply:
Agreed! The function name comes from the collector code where over there they use a featureflag called 'Normalize'. That doesn't make much sense in our context :)